tests/infrastructure/instance_types/test_common_vm_instancetype.py

```diff
@@ -22,9 +16,10 @@ def test_common_instancetype_vendor_labels(base_vm_cluster_instancetypes):
     assert_mismatch_vendor_label(resources_list=base_vm_cluster_instancetypes)
 
 
+@pytest.mark.hugepage
```

Comment:
Does this require fully capable cluster?

Reply:
if we have huge pages , we are good.

Reply:
you are not marking them with special_infra. Devops won't allocate fully capable clusters for tests that are `not` using special_infra marker.

Reply:
IIRC,
**For BM Clusters:** hugepages is configured as part of Deploy jobs
**For PSI Clusters:** hugepages is configured a part of Test jobs, something to sync up with devops-qe for uniformity?

Reply:
I just confirmed with Denis, from DevOps QE about this and he confirmed the same for hugepages.

Reply:
if i have this marker, i assume this test will only be run in a pre-configured hugepage cluster and that's enough for this test.

Reply:
There would be no test job with just hugepage marker. This was discussed in programmatic skip meeting last week. Only two test jobs currently 1) with special_infra marker 2) without special_infra marker.